internal/flink/command_artifact_create.go

```diff
@@ -37,13 +36,15 @@ func (c *command) newCreateCommand() *cobra.Command {
 		),
 	}
 
-	cmd.Flags().String("artifact-file", "", "Flink artifact JAR file.")
+	cmd.Flags().String("artifact-file", "", "Flink artifact")
+	cmd.Flags().String("runtime-lang", "java", "Flink artifact language runtime python/java.")
```

Comment:
How about `--runtime-language` for clarity?

Reply:
Done updated.